### ISSUE TITLE
Omnibus 4a

### ISSUE THREAD

**Issue by JamesB192** [JamesB192/Shadowrun-Unlimited-rip#17]
Thanks so much for keeping this thing alive! It's addictive as hell and I'm so glad it's not dead.

# Here are some bugs... Let me know if you want me to go into more detail!

* Sometimes on runs and in the ranged arena, NPC's seem to either spawn without guns or simply don't equip them. They just run headlong at the player and throw punches.

* ~~Mitsuhama statue mission is a perfect example: I opened the security office door on the lobby floor and they immediately started running around like chickens with their heads cut off. I had to chase them around the whole floor as they ran around randomly... they didn't even try to target me.~~ Yay, they was mostly _blind_, except for eyes in the back of their heads.

* ~~This MAY have something to do with the "Change employee schedule" hack but that wouldn't explain it happening in the ranged arena.~~ It _should_ not.

* ~~Wild Yak firebombing creates some sort of extra playable character with no portrait stuck in an offscreen space and which can't move. It's not game-breaking but it is jarring to have to manually end each turn.~~ This is intentional. There are three ways to avoid having the invisible runner spawn. You could hire enough runners that the party count variable is two or more, you could enable risk permadeath in the commlink, or you can bludgeon someone until it goes away by fixing the bug where death is permanent if there are no other runners on your team.

* ~~What is a bug is that my first try at the Wild Yak one of the guys in the last room glitched off the map so combat couldn't end.~~ I looked but I am not seeing the source of this one.

* Matrix is buggy in a variety of ways and they're hard to pin down. Sometimes after failing a node, the UI glitches out and it's impossible to do anything. One time I went into the character menu and found Frosty's character portrait and one other that I can't remember.

* In some mission matrix runs the "Management Files" node will double-fire after getting Paydata and employee schedules changed simultaneously. Every subsequent use of the node costs -2 AP and checks for paydata twice.

* ~~The Bodybuilding trainer in the Puyallup safehouse bugs out each time by giving tons of extra hit points after multiple successive uses. At one point, I had 200+ HP from a base of 50. This seems to reset on heals buuuut....~~

* Heals don't seem to trigger after some missions, leaving the player with a wound in non-combat play that doesn't automatically heal except in certain circumstances.

# Unclear if these are bugs:

* After the elven heist mission, my next step is to talk to Piper. He says he'll give contact info for a new informant, but that never happens. Is that just the hard stop of the story so far?

* ~~Some paydata files (exceptionally low number ones) can't be sold to data brokers. If that's WAI then that's really annoying because I have to go into Asset Management to delete them. There should be an option in data broker convo to delete worthless files. It should go without saying that some players might not even know that the option to delete them exists.~~

* ~~Buying data storage doesn't seem to have an upper limit anymore and there's no dialogue explaining how much you can have with a given deck. I bought something like 1000 MP worth of it back when I had the Sony and it wasn't until I got the Fuchi that I saw the message saying "Well I guess I can't fit any more storage space." The Renraku didn't give me that message either.~~

* Having the Melee Arena Renraku ambush mission active makes it impossible to complete any other mission or do random Matrix grinds... which makes a player stuck and screwed if they can't get past it. Even after I successfully completed the mission, the job I took from Frosty during it was permanently glitched and I had to bail on the run to take new jobs.

* Data nodes locked behind CPU's that can only be reached from inside mission buildings during a run do nothing when interacted with. I have yet to see a Red/Orange data node in this build and, as a tricked-out decker, this would be a really nice place for them.

# Not bugs, but annoying:

* ~~In the Renraku melee arena ambush, the hostile NPC's all go first and turn you + your team into swiss cheese immediately. First I went in solo and got destroyed so hard that, immediately after using a DocWagon revive, it prompted me for a SECOND revive which I didn't have. I went in with a full team and they got grenaded + shot to hell immediately. I then went in a third time fully kitted out and, after losing half the team immediately, I managed to use them as cannon fodder long enough to get to cover and slowly grind down the bullet sponge enemies while spamming medkits.~~ Tweaked ambush. you should go first now and they should scale from a lower base.

* ~~There are a bunch of typos in the dialogue for everything after the first Vigor mission. I'm a professional writer and I'd be happy to copy-edit if that'd be helpful.~~ I think _mostly_ fixed. There a few convos and lots of assets I haven't got to yet.

* ~~The Paydata nodes are a mess and it would probably be easier on the code if they weren't activated multiple times by design with -1AP's every time. Maybe replace them with a dialogue tree? And balance it all out by increasing alarm heat the more you stay in dialogue?~~

* ~~The paydata and mission reward values should honestly be reversed. Missions should bring in 5k-20k and files should bring in 500-2000, not vice versa. Is the current system the way it is in PNP? If so, that's really messy and unbalanced.~~

* Also, while I appreciate the detail of having different programs for different decking actions... it's really just an overcomplicated money sink that should all fit under the "Decking" skill the player already pays for. It doesn't really add anything to the gameplay, it just subtracts from enjoyment.

* It'd be really nice if there were hostile deckers inside some Matrix runs... it'd mean there'd be a point to having that Black ICE ability or Killjoy or anything like that.

* It would also be sweet to be ambushed in the Hub by hostile deckers after attracting a certain amount of heat or notoriety.

* I REALLY don't like the fact that there are repeatable abilities that heal X HP instead of most recent wound. That's not a game-breaker, but it is an unbalanced and weird departure from the base game rules.

# New bugs in intro escort missions:

* ~~Decker hacking vidphones glitches out sometimes by refusing to start decking unless right on top of the jack-in spot and has to be kited into position. One time, the dialogue to start decking happened then kept repeating until I kited her into position and restarted it.~~

* ~~Escort suit to Downtown mission locked me into permanent combat when one of the three thugs apparently spawned or glitched off the map.~~

~~This may only be happening in Downtown, but I've also noticed that often a Lonestar trigger will double-fire, leading to two different cops having identical conversations with you. I'm running an adept now.~~ Oops, my bad...

Another gripe... ~~the fact that Greely doesn't have ANY mid-tier melee/unarmed weaponry really blows.~~ Greely now has a few things.

Also you mentioned in a previous thread that there's a drug dealer that you can buy from. Whare?

**Comment by AzyWng** [comment]
I'm really not sure how to contact anyone about this, so I'm just writing it here...

I don't know why, but I can't seem to install this content pack at all. Whenever I try, I get errors telling me about missing files. I've already followed the instructions.
![WcIjP4zXjW](https://user-images.githubusercontent.com/45751345/106370721-633e5780-6322-11eb-8073-af08d141bb57.png)

I downloaded the repository from the stableish branch, and am using GOG's version of Shadowrun Dragonfall: Director's Cut. Is there a file I'm forgetting to install, or some other mistake I'm making?

**Comment by JamesB192** [comment]
Those look like missing files from the HBS content packs. I suppose you have followed the initial setup at https://shadowrun.gamepedia.com/Getting_Started .

**Comment by AzyWng** [comment]
Oh, I actually hadn't. Looking through the Getting Started page now and I have to ask - Do I need to have the other Shadowrun games in addition to Dragonfall in order to play this mod?

**Comment by JamesB192** [comment]
No, you would only need them if you wee teing to port something to the pack from that game or port the game to that game.

**Comment by AzyWng** [comment]
All right. After following the instructions on the Getting Started page, the mod loads.
Thanks for helping me. I'll let you know if I run into any other issues.
